core/nodejsActionBase/launcher.js

```diff
@@ -27,9 +29,16 @@ try {
     } catch (e) {}
   })();
 
-  const lambdaCompat = process.env.__OW_LAMBDA_COMPAT === undefined ? false : process.env.__OW_LAMBDA_COMPAT.toLowerCase() === 'true' && NodeActionLambdaRunner !== undefined;
   const handler = eval('require("./##MAIN_FILE##").##MAIN_FUNC##') // Will be replaced in the compile script with the correct main.
-  const runner = lambdaCompat === false ? new NodeActionRunner(handler) : new NodeActionLambdaRunner(handler);
+
+  // Deem function to be Lambda if env var is provided (for backwards
+  // compatibility) or if the function signature is detected to be one for
+  // Lambda.
+  const lambdaCompat = ((process.env.__OW_LAMBDA_COMPAT !== undefined && process.env.__OW_LAMBDA_COMPAT.toLowerCase() === 'true')
```

Comment:
nit: Should we move this check inside the `isLambda` check? Since you've written tests for it, we can make sure that the logic is actually sound. Gut feeling says that something like `!!process.env.__OW_LAMBDA_COMPAT` should do the trick here too, but my Node.js proficiency is long gone.

Reply:
Agree. Will change function name to `useLambdaRunner` and expand its scope. It'll take into account env vars and the signature detection.